Assignment 5.Rmd

```diff
@@ -75,3 +79,16 @@ D2$prediction <- predict(score_ctree, D2)
 ``` 
 ## Part III
 Compare the predicted advice with the actual advice that these students recieved. What is the difference between the observed and predicted results?
+
+```{r}
+
+D1$prediction <- predict(score_ctree, D1)
+
+D1$action <- ifelse(D1$action ==1, "intervene", ifelse(D1$action ==2, "monitor", "no action"))
+
+sum(D1$action==D1$advice)/378
+sum(D1$prediction==D1$action)/378
+sum(D1$prediction==D1$advice)/378
```

Comment:
Not quite, all the students in the new sample received a score of 1, therefore none of them require assistance according to the new model. The accuracy rate should therefore be 58%.